xbmc/ServiceBroker.cpp

```diff
@@ -16,30 +16,43 @@
 
 using namespace KODI;
 
-std::unique_ptr<CLog> CServiceBroker::m_logging;
+
+CServiceBroker::CServiceBroker() :
+    m_pGUI(nullptr),
+    m_pWinSystem(nullptr),
+    m_pActiveAE(nullptr),
+    m_pSettingsComponent(nullptr),
+    m_decoderFilterManager(nullptr)
```

Comment:
use default member initializers

Reply:
No. Not in this case. That would split initialization across two files making it impossible to look in one place and know what's going on. If that's in the coding guidelines for non-default constructors that are in the .cpp file, it needs to be changed there.

Reply:
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#114-default-member-initialization says "Use default member initialization instead of initializer lists or constructor assignments _**whenever it makes sense**_." (emphasis added). It doesn't make sense in this case.

Reply:
Also note: https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#116-constructor-initialzation-lists